service/tablet_allocator.cc

```diff
@@ -1084,6 +1084,35 @@ class load_balancer {
         return tablet_count;
     }
 
+    std::tuple<schema_ptr, const tablet_aware_replication_strategy*> get_schema_and_rs(table_id table) {
```

Comment:
nit: in git log s/readibility/readability/

Also, it'd probably be better to move this patch earlier in the series before any functional changes are made.